### ISSUE TITLE
Alerting + group by time low limit

### ISSUE THREAD

**Issue by miniskipper** [grafana/grafana#7051]
When using the low limit for the group by time (>1m) in the metric and assigning an alert, there is following error:

```
{
  "error": "error parsing query: found >, expected identifier, string, number, bool at line 1, char 131",
  "message": "error parsing query: found >, expected identifier, string, number, bool at line 1, char 131"
}
```

Cause is, that the statement is sent to the influx-db the following way:	

`SELECT mean("response_time") FROM "net_response" WHERE "host" =~ /^.*_ts3$/ AND time > 1482361200s and time < now() GROUP BY time(>1m), "host" fill(null)`

Version 4.0.2 (commit: v4.0.2)
InfluxDB v1.1.1 (git: master e47cf1f2e83a02443d7115c54f838be8ee959644)

When removing the low limit for the group by time, the error is gone.

**Comment by bergquist** [comment]
@miniskipper are you using `time(auto)` or `time($interval)`?

time(auto) have a bug that have been fixed but not released (4.0.3 or 4.1.0). But its included in 4.1.0-beta1.

**Comment by miniskipper** [comment]
I was using the auto setting. Will try beta version.

**Comment by bergquist** [comment]
Closing this since you haven't reported back. If the issues still exist. Get back to me in this issue and I will reopen it.

**Comment by Artanicus** [comment]
Am I reading this correctly that this issue should be fixed already in 4.1.2? I'm running 4.1.2 and still suffering from this:
InfluxDB Error Response: error parsing query: found >, expected identifier, string, number, bool at line 1, char 137

- Using fill(auto) and $interval >2s
- works fine until some time after adding alerting to the query
- other queries in the same graph work fine if I hide the query with an alert attached
- query: SELECT mean("value") FROM "load_longterm" WHERE "host" = 'phrenos' AND $timeFilter GROUP BY time($interval) fill(null)
